doc/configuration.rst

```diff
@@ -1117,54 +1117,58 @@ See the Sphinx-Gallery `Sphinx configuration file
 <https://github.com/sphinx-gallery/sphinx-gallery/blob/master/doc/conf.py>`_
 for an example that uses the `public Binder server <https://mybinder.org>`_.
 
-.. _jupyterlite
-Generate Jupyterlite links for gallery notebooks (experimental)
+.. _jupyterlite:
+
+Generate JupyterLite links for gallery notebooks (experimental)
 ===============================================================
 
 Sphinx-Gallery automatically generates Jupyter notebooks for any examples built
-with the gallery. `JupyterLite <https://jupyterlite.readthedocs.io>`_ makes it
+with the gallery. `JupyterLite <https://jupyterlite.readthedocs.io>`__ makes it
 possible to run an example in your browser. TODO mention difference with Binder?
```

Comment:
```suggestion
possible to run an example in your browser. This is different from Binder links where
the notebook is run on a server. By running in the reader's browser the startup time
can be much lower compared to Binder. However some libraries or parts of some
libraries do not yet work in the browser.
```

How about something like this? Just an idea I had as I was reading the diff (didn't take long to write and maybe you find it useful)

Reply:
Nice thanks a lot! (I thought I tackled all the TODOs left but apparently not :wink:)